draft-ietf-tls-esni.md

```diff
@@ -1229,11 +1223,11 @@ can be used to test the encrypted SNI value of specific ClientHello messages.
 ~~~
 {: #tls-resumption-psk title="Message flow for resumption and PSK"}
 
-ECH mitigates against this attack by requiring servers not mix-and-match
-information from the inner and outer ClientHello. For example, if the server
-accepts the inner ClientHello, it does not validate binders in the outer
-ClientHello. This means that ECH PSKs are used within the HPKE encryption
-envelope.
+ECH mitigates against this attack by (1) prohibiting the "pre_shared_key" 
+extension in outer ClientHello and (2) requiring servers ignore this extension 
+when processing the outer ClientHello. Thus, if the server accepts the inner 
+ClientHello, it only validates binders in the inner ClientHello. This means 
+that ECH PSKs are used within the HPKE encryption envelope.
```

Comment:
Is "ECH PSK" an obsolete term?

Reply:
We use it when noting tickets/PSKs associated with an ECH accepted connection.